src/analysis/retail/monk/mistweaver/modules/spells/ThunderFocusTea.tsx

```diff
@@ -168,6 +193,58 @@ class ThunderFocusTea extends Analyzer {
     return <DonutChart items={items} />;
   }
 
+  /** Guide subsection describing the proper usage of TFT */
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <b>
+          <SpellLink id={TALENTS_MONK.THUNDER_FOCUS_TEA_TALENT.id} />
+        </b>{' '}
+        is an important spell used to empower other abilities. It should be used on cooldown at all
+        times and the spell that you use it on depends on your talent selection. If you have{' '}
+        <SpellLink id={TALENTS_MONK.SECRET_INFUSION_TALENT} />, then you should always use{' '}
+        <SpellLink id={TALENTS_MONK.THUNDER_FOCUS_TEA_TALENT.id} /> on{' '}
+        <SpellLink id={TALENTS_MONK.ESSENCE_FONT_TALENT} /> (with high{' '}
+        <SpellLink id={TALENTS_MONK.UPWELLING_TALENT.id} /> stacks) or{' '}
+        <SpellLink id={TALENTS_MONK.RENEWING_MIST_TALENT.id} />. If you do not have{' '}
+        <SpellLink id={TALENTS_MONK.SECRET_INFUSION_TALENT.id} />, then always use it on{' '}
+        <SpellLink id={TALENTS_MONK.RISING_SUN_KICK_TALENT.id} /> (with{' '}
+        <SpellLink id={TALENTS_MONK.RISING_MIST_TALENT.id} />
+        ), otherwise use it on <SpellLink id={TALENTS_MONK.RENEWING_MIST_TALENT.id} />
+      </p>
+    );
+    const data = (
+      <div>
+        <RoundedPanel>
+          <strong>
+            <SpellLink id={TALENTS_MONK.THUNDER_FOCUS_TEA_TALENT.id} /> cast efficiency
+          </strong>
+          <div>
+            {this.subStatistic()} <br />
+            <small>
+              Green indicates a correct <SpellLink id={TALENTS_MONK.THUNDER_FOCUS_TEA_TALENT} />{' '}
+              cast, while red indicates a bad cast.
```

Comment:
this should either be "correct/incorrect" or "good/bad", not "correct/bad" or "good/incorrect".

i would generally avoid "correct/incorrect" personally (also applies to the tooltips above) but i'm not a stickler on that. personal preference